cookbooks/ascii_perturbation/ascii_perturbation.prm

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
Can you add a brief description (2-4 sentences) of the benchmark at the top of the PRM file?